cumulus/parachains/runtimes/assets/asset-hub-kusama/src/xcm_config.rs

```diff
@@ -540,6 +540,16 @@ pub type XcmRouter = WithUniqueTopic<(
 #[cfg(feature = "runtime-benchmarks")]
 parameter_types! {
 	pub ReachableDest: Option<MultiLocation> = Some(Parent.into());
+	// Relay/native token can be teleported between AH and Relay.
+	pub TeleportableAssets: Option<(MultiAssets, MultiLocation)> = Some((
+		MultiAsset { fun: Fungible(10), id: Concrete(Parent.into()) }.into(),
+		Parent.into(),
+	));
+	// We can reserve transfer some local token to Relay.
+	pub ReserveTransferableAssets: Option<(MultiAssets, MultiLocation)> = Some((
+		MultiAsset { fun: Fungible(10), id: Concrete(Here.into()) }.into(),
```

Comment:
Wait, let's get this straight. If you want to specify the relay chain native token, then this should really be `Parent`, not `Here`. We shouldn't try to have aliases of the same asset, even though in our heads we know that AH uses KSM/DOT natively.

Reply:
Yes, I agree this is confusing. For this benchmarks test I was actually aiming at some `local-reserve` asset and used simplest location id `Here`, but you're right that it can be confused with trying to target the native token specifically.

I can't use relay chain native token here because that's teleportable so it will always be teleported. I'll try to create/use some other dummy _local_ asset which should not be confused with native asset. Maybe smth like: `MultiAsset { fun: Fungible(10), id: Concrete(GeneralIndex(42).into()) }`

Reply:
fixed